lib/robots/dor_repo/gis_delivery/load_raster.rb

```diff
@@ -14,14 +14,7 @@ def perform_work
           rootdir = GisRobotSuite.locate_druid_path bare_druid, type: :workspace
 
           # determine whether we have a Raster to load
```

Comment:
missed this until i saw the similar #696, but seems like this comment could use a tweak for accuracy (or could be removed, since the method name is now pretty self-explanatory, and requires no file loading as a prior step)